JustFakeIt/project.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": "1.0.0",
+    "authors": [ "JUST EAT plc" ],
+    "copyright": "JUST EAT plc",
+    "title": "JustFakeIt",
+    "description": "An InProcess HTTP server which can be mocked and asserted against to allow for full stack HTTP testing",
+    "buildOptions": {
+        "warningsAsErrors": false,
+        "platform": "anycpu",
+        "languageVersion": "csharp6",
+        "debugType": "portable"
+    },
+    "packOptions": {
+        "owners": [ "JUST EAT plc" ],
+        "summary": "An InProcess HTTP server which can be mocked and asserted against to allow for full stack HTTP testing",
+        "tags": [ "owin", "http", "mock", "testing" ],
+        "repository": {
+            "type": "git",
+            "url": "https://github.com/justeat/JustFakeIt.git"
+        },
+        "requireLicenseAcceptance": true,
+        "licenseUrl": "https://github.com/justeat/JustFakeIt/blob/master/LICENSE.md",
+        "projectUrl": "https://github.com/justeat/JustFakeIt"
+    },
+    "configurations": {
+        "Debug": {},
+        "Release": {
+            "buildOptions": {
+                "optimize": true
+            }
+        }
+    },
+    "frameworks": {
+        "netstandard13": {
+            "dependencies": {
+                "Microsoft.AspNetCore.Http.Abstractions": "1.0.0",
+                "Microsoft.AspNetCore.Owin": "1.0.0",
```

Comment:
See [the docs](https://docs.asp.net/en/latest/fundamentals/owin.html#running-owin-middleware-in-the-asp-net-pipeline)